.github/ISSUE_TEMPLATE/pull_request.md

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
I think pull request templates should be under `PULL_REQUEST_TEMPLATE` directory or just named as so under `.github`

```
.github/PULL_REQUEST_TEMPLATE/pull_request.md
.github/PULL_REQUEST_TEMPLATE.md
```

The top-level metadata syntax is probably also not needed